bin/oref0-setup.sh

```diff
@@ -169,6 +173,19 @@ if [[ -z "$DIR" || -z "$serial" ]]; then
       # Force uppercase, just in case the user entered ww
       radio_locale=${radio_locale^^}
 
+      # check if user has a TI USB stick and a WorldWide pump and want's to reset the USB subsystem during mmtune if the TI USB fails
+      ti_usb_ww0="no" # assume you don't want it by default
+      ti_usb_ww1=""
+      if [[ $radio_locale =~ ^WW$ ]]; then
+        echo "If you have a TI USB stick and a WW pump, you might want to reset the USB subsystem if it can't be found during a mmtune process"
+        read -p "Do you want to reset the USB system in case the TI USB stick can't be found during a mmtune proces? Use y if so. Otherwise just hit enter: " -r
+        if [[ $REPLY =~ ^[Yy]$ ]]; then
+          ti_usb_ww="yes"
+        else
+          ti_usb_ww="no" 
```

Comment:
Having both upper and lowercase versions of the same variable name is very confusing.  Can we rename them so it's clear what's being done?